src/Apps/Order/redirects.tsx

```diff
@@ -175,6 +181,7 @@ export const redirects: RedirectRecord<{
         goToReviewIfOrderIsPending,
         goToShippingIfShippingIsNotCompleted,
         goToPaymentIfPaymentIsNotCompleted,
+        goToRespondIfAwaitingBuyerResponse,
```

Comment:
Wooo nice!!

Curious why the route below got updated? 

Also, can we add a test for this one? 😄

Reply:
I'm not sure why the route below got updated 🤔 pretty much every time I touch my local environment it changes.

I'm actually not sure where tests for these live -- it doesn't seem like any of the other redirects are tested?

Reply:
All of the tests live in here! https://github.com/artsy/reaction/blob/master/src/Apps/Order/__tests__/OrderApp.test.tsx

Reply:
Ah, thanks!